parser/parser.y

```diff
@@ -12889,6 +12892,11 @@ CommentOrAttributeOption:
 		$$ = &ast.CommentOrAttributeOption{Type: ast.UserAttributeType, Value: $2}
 	}
 
+|  "RESOURCE" "GROUP" stringLit
```

Comment:
I prefer to declare `"RESOURCE" "GROUP" stringLit` with a new term `ResourceGroupOption`, and add `ResourceGroupOption` into `CreateUserStmt` and `AlterUserStmt` after `CommentOrAttributeOption`, like
```
AlterUserStmt:
	"ALTER" "USER" IfExists UserSpecList RequireClauseOpt ConnectionOptions PasswordOrLockOptions CommentOrAttributeOption ResourceGroupOption
```

Reply:
In my implementation, "resource group" is just one attribute for user, like email. which is stored in User_attributes column of mysql.user.

Reply:
So can we just use 'alter user ... attribute ...' without provide an extra new syntax?

Reply:
No. Resource group might be a frequently used attribute. So, we introduce new syntax for it following the pattern of comment.